pyproject.toml

```diff
@@ -1,7 +1,92 @@
+# pyproject.toml
+
 [build-system]
-requires = ["setuptools<69", "wheel", "numpy<2.0", "pyparsing<=3.1.1", "scipy"]
+# These are the build-time dependencies. `setuptools` is the build backend,
+# and `numpy` is required for its C headers when compiling the extensions.
+requires = ["setuptools>=61.0", "numpy>=1.10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+# All static project metadata goes here.
+name = "ProDy"
+dynamic = ["version"]  # The version is read from prody/__init__.py
+description = "A Python Package for Protein Dynamics Analysis"
+readme = "README.rst"
+requires-python = ">=3.10" # Based on the logic in your setup.py
+license = { text = "MIT License" }
+keywords = [
+    "protein", "dynamics", "elastic network model",
+    "Gaussian network model", "anisotropic network model",
+    "essential dynamics analysis", "principal component analysis",
+    "Protein Data Bank", "PDB", "GNM", "ANM", "SM", "PCA",
+]
+authors = [
+    { name = "James Krieger", email = "jamesmkrieger@gmail.com" },
+    { name = "Karolina Mikulska-Ruminska" },
+    { name = "She Zhang" },
+    { name = "Hongchun Li" },
+    { name = "Cihan Kaya" },
+    { name = "Ahmet Bakan" },
+    { name = "and others" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+    "Topic :: Scientific/Engineering :: Chemistry",
+]
+
+# Runtime dependencies
+dependencies = [
+    "numpy>=1.10",
+    "biopython",
+    "pyparsing<=3.1.1",
+    "scipy",
+]
+
+[project.urls]
+Homepage = "http://www.csb.pitt.edu/ProDy"
```

Comment:
This needs updating to http://www.bahargroup.org/prody/. That link doesn't work

Also, please add the following
Documentation = "http://www.bahargroup.org/prody/tutorials.html"
Source = "https://github.com/prody/ProDy"
CondaForge = "https://conda.anaconda.org/conda-forge"

Reply:
Added.